README.md

```diff
@@ -447,6 +497,12 @@ path:
 },
 ```
 
+Then:
+
+```
+npm run loc
```

Comment:
For using this command, Shouldn't we define `loc` under the script key in `package.json`?

Reply:
Look 5 lines before that

Reply:
oh. I see. I didn't see the lines above because they were collapsed.